.github/scripts/install.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+set -ev
```

Comment:
Instead of bash, I suggest to use cmake script. 

### FetchHoloPlayCore.cmake

A file called `FetchHoloPlayCore.cmake` that would:
* set relevant variable in parent scope
* output path of variable so that calling processing can gather values

See https://github.com/KitwareMedical/SlicerLookingGlass/blob/master/SuperBuild/External_HoloPlayCore.cmake for re-using

We would then update `SlicerLookingGlass` to also make use of this convenience script.

### FetchVTKExternalModule.cmake

See https://github.com/KitwareMedical/SlicerIMSTK/blob/master/FetchVTKExternalModule.cmake

Reply:
This sounds good. I am going to look into this.